cli/tests/pcluster/templates/test_cluster_stack.py

```diff
@@ -27,11 +27,6 @@
 @pytest.mark.parametrize(
     "config_file_name",
     [
-        "slurm.required.yaml",
-        "slurm.full.yaml",
-        "awsbatch.simple.yaml",
-        "awsbatch.full.yaml",
-        "byos.required.yaml",
```

Comment:
probably you didn't mean to remove these

Reply:
right, removed by mistake.